policyengine_us/parameters/gov/states/la/tax/income/exemptions/aged.yaml

```diff
@@ -0,0 +1,19 @@
+description: Louisiana provides an exemption for aged head or spouse.
+
+brackets:
+  - threshold:
+      2001-01-01: 65
+    amount:
+      2001-01-01: 6_000
+
+metadata:
+  reference:
+    - title: "Annual retirement or disability income; exemption from taxation, part A"
+      href: https://www.legis.la.gov/legis/Law.aspx?d=102133
```

Comment:
this law specifies that this is a retirement exemption -> for pension income 
it also has a disability part -> please read the law and look through the tax form instructions to find the right passage